.circleci/config.yml

```diff
@@ -2316,17 +2316,7 @@ workflows:
       not:
         equal: [scheduled_pipeline, << pipeline.trigger_source >>]
     jobs:
-      # Wait for approval on the release
-      - hold:
-          type: approval
-          filters:
-            tags:
-              only: /^(da-server|cannon|ufm-[a-z0-9\-]*|op-[a-z0-9\-]*)\/v.*/
-            branches:
-              ignore: /.*/
       - initialize:
-          requires:
-            - hold
```

Comment:
Hmm - why are we doing this?  I have definitely accidentally pushed the wrong tag up before - seems like this makes pushing tags riskier if we then automatically kick off a release.